package.json

```diff
@@ -1,14 +1,30 @@
 {
   "description": "Desktop application for Centar Srce",
   "license": "MIT",
+  "name": "srce_desktop_app",
   "repository": {
     "type": "git",
     "url": "https://github.com/nsoss/srce-desktop-app.git"
   },
   "scripts": {
-    "start": "concurrently \"npm run react-start\" \"wait-on http://localhost:3000 && electron .\"",
+    "start": "concurrently \"npm run react-start\" \"wait-on http://localhost:3000 && electron . \" ",
     "react-start": "react-scripts start",
-    "build": "react-scripts build"
+    "build": "react-scripts build",
+    "pack": "electron-builder --dir",
+    "dist": "electron-builder",
+    "build-electron": "robocopy . build index.js ",
+    "build-src": "cd build && mkdir src && cd src && mkdir database && cd .. && cd .. && robocopy src/database build/src/database srce.db /S",
+    "package": "electron-builder build  --win -c.extraMetadata.main=build/index.js --publish never",
```

Comment:
Maybe consider adding a win suffix, something like `package:win` so that when you add package management for *nix you can do a `package:osx` or `package:linux` (if that ever comes up in the future)